onyo/cli/tests/test_set.py

```diff
@@ -121,7 +121,7 @@ def test_set_multiple_assets(repo: OnyoRepo,
 @pytest.mark.parametrize('no_assets', non_existing_assets)
 def test_set_error_non_existing_assets(repo: OnyoRepo,
                                        no_assets: list[str]) -> None:
-    """Test that `onyo set KEY=VALUE <asset>` errors correctly for:
+    r"""Test that `onyo set KEY=VALUE <asset>` errors correctly for:
```

Comment:
Not caused by you, since it was like this before: but I want to note that it should be `onyo set --keys KEY=VALUE --path <asset>` (path might be called asset now?) because the flags need to be stated explicitely.

This is probably a legacy error from way back when the flags were not required for `onyo set`.

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.